web/src/pill_typeahead.ts

```diff
@@ -86,10 +87,12 @@ export function set_up_stream(
         type: "contenteditable",
     };
     opts.help_on_empty_strings ??= false;
+    opts.hide_on_empty_after_backspace ??= true;
```

Comment:
Why is this default `true`?

Reply:
Ah yeah I think it should be default false, good catch

Reply:
I guess there is no good reason for that. I made it false.